go.mod

```diff
@@ -18,6 +18,8 @@ require (
 	k8s.io/klog/v2 v2.130.1
 )
 
+require github.com/RobinUS2/golang-moving-average v1.0.0 // indirect
```

Comment:
This could do with tidying; I don't see a reason it should be in its own block, and it isn't indirect ?